wolfcrypt/src/random.c

```diff
@@ -2741,14 +2745,26 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         }
     #endif /* HAVE_INTEL_RDSEED */
 
+#ifdef WIN_REUSE_CRYPT_HANDLE
```

Comment:
`OS_Seed` already contains `ProviderHandle`,  wouldn't it be enough _not_ to release it with `CryptReleaseContext` and init only when not initialised?  This will avoid global static

Reply:
There could be multiple OS_Seed provider handles and without a global static there would be no way to properly cleanup handles.